src/gql.ts

```diff
@@ -1,15 +1,20 @@
-import {InMemoryCache, NormalizedCache} from 'apollo-cache-inmemory';
+import {InMemoryCache, IntrospectionFragmentMatcher, NormalizedCache} from 'apollo-cache-inmemory';
 import ApolloClient from 'apollo-client';
 import {ApolloQueryResult} from 'apollo-client/core/types';
 import {WatchQueryOptions} from 'apollo-client/core/watchQueryOptions';
 import {HttpLink} from 'apollo-link-http';
 import fetch from 'node-fetch';
 import {promisify} from 'util';
 
+const schema = require('../github-schema.json');
+
 export class GitHub {
   private apollo: ApolloClient<NormalizedCache>;
 
   constructor(uri = 'https://api.github.com/graphql') {
+    const fragmentMatcher = new IntrospectionFragmentMatcher(
```

Comment:
Maybe a quick comment on why this is needed since it's a little odd looking?

Reply:
Done.